app/js/components/orderEntry/AllOrders.jsx

```diff
@@ -6,11 +6,24 @@ import SortAndFilter from './SortAndFilter';
 import Paginate from './Paginate';
 import fetchOrders from '../../actions/fetchOrders';
 
+const determineUrl = (links, action) => {
```

Comment:
What does determine url mean?

Reply:
The name is not clear enough. Determine url for what? For fetching data from the server? Or what? Can you use a name that does not need more explanation to figure out what the method is used for?